eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -223,6 +223,14 @@ stages:
                           PRLabels: "auto-merge"
                           CloseAfterOpenForTesting: '${{ parameters.TestPipeline }}'
 
+  - stage: SmokeTest_Release_Packages
```

Comment:
Should we resolve the intermittent failures that occur from the China cloud due to the linked template in test resources before enabling this on release?   Daniel was investigating at one point, but I don't know where that left off.

Reply:
Yes I think that's a good idea. Adding a flaky test to the release pipeline would not be good. As far as I see, there is the outstanding iothub issue with gov cloud, and the github raw content intermittent 404 for China cloud.

Reply:
Sounds about right.   

Not sure how helpful it is, but a bit of context on the linked template:  we had started off without the linked template, but there were concerns raised about duplication and the need to keep multiple templates in-sync, so we shifted to the linked approach.    I wonder if we want to reevaluate the cost/benefit there having seen how frequently China struggles to resolve the link.

Reply:
Yeah, though if it changes frequently, having it be checked in could potentially exacerbate some of the git clone problems we're seeing as a general result of all the test recordings, etc. I'm not sure how much drift/how many files there are though.

Reply:
In this case, it would be a copy/paste of [these](https://github.com/Azure/azure-sdk-for-net/blob/master/samples/iothub-connect-to-eventhubs/sample-resources.json) into [here](https://github.com/Azure/azure-sdk-for-net/blob/master/common/SmokeTests/SmokeTest/test-resources.json).    Neither changes frequently, in my experience.   Though, the concern of "if we change a sample, it's likely we'll forget to reflect that change to the smoke tests" is valid.

Outside of doing preprocessing on the ARM templates and trying to merge all the linked templates into the main prior to deploy.... I don't have a good idea of another approach.

Reply:
It looks like there is a private preview for `relativePath` which will support local template linking: https://github.com/microsoft/vscode-azurearmtools/issues/588. Not sure what the release schedule for it is, I'll dig.

Reply:
@jsquire the dev working on it estimates end of January, so I'm thinking to copy over the templates for now, and make a work item to remove them once the file URI support gets added: https://github.com/microsoft/vscode-azurearmtools/issues/588#issuecomment-751318211

Reply:
@jsquire PR for temporary inlining: https://github.com/Azure/azure-sdk-for-net/pull/17758